mod/file/views/default/icon/object/file.php

```diff
@@ -9,35 +9,38 @@
  * @uses $vars['link_class'] Optional CSS class added to link
  */
 
-$entity = $vars['entity'];
-
-$sizes = array('small', 'medium', 'large', 'tiny', 'master', 'topbar');
```

Comment:
anything that's not `small`, `medium`, `large` would return a `large` thumbnail. Not sure why we have `tiny` and `topbar` here. Let's remap `master` to `large`, and rest to `small`?